log/sequencer.go

```diff
@@ -189,10 +191,22 @@ func (s Sequencer) sequenceLeaves(mt *merkle.CompactMerkleTree, leaves []*trilli
 		}
 		// The leaf has now been sequenced.
 		leaves[i].LeafIndex = seq
```

Comment:
nit: Use `leaf` instead of `leaves[i]`. Same throughout.

Reply:
done